internal/controllers/utils/utils.go

```diff
@@ -298,12 +299,35 @@ func GetSearchURL(ctx context.Context, c client.Client) (string, error) {
 	return fmt.Sprintf("https://%s.%s.svc.cluster.local:%d", service.Name, service.Namespace, service.Spec.Ports[0].Port), nil
 }
 
+// GetACMNamespace attempts to find the namespace being used by ACM by looking up the Search API service.
+func GetACMNamespace(ctx context.Context, c client.Client) (string, error) {
+	service, err := GetSearchAPI(ctx, c)
+	if err != nil {
+		return "", err
+	}
+	return service.Namespace, nil
+}
+
+// GetServerDatabasePasswordName retrieves name of the environment variable used to store the server's database password
+func GetServerDatabasePasswordName(serverName string) (string, error) {
+	if name, ok := DatabaseNameMappings[serverName]; ok {
```

Comment:
There are already `utils.AlarmsPasswordEnvName`and `utils.ResourcesPasswordEnvName`. Cleaner to do if serverName == InventoryAlarmServerName return utils.Alarms......

No need to return errors or have a name mapping

Reply:
This is to lookup the name of the logical database for each server.  The variables that you referenced hold the passwords for those servers.

Reply:
you create a secret called fmt.Sprintf("%s-passwords", utils.InventoryDatabaseServerName). That secret has as keys the constants I mention. 

Now, here you are trying to find the name of the key related to the server which are the same constants.

Reply:
I can change it to `if` statements but we'll see need to return an error if the server name doesn't match one of the 2 supported values.